assets/install-scripts/install-teleport-tests/docker-compose.yml

```diff
@@ -0,0 +1,102 @@
+version: "3.8"
```

Comment:
When I run docker compose I get an error: `the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion` (docker compose 2.29.6)